index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="styles/style.css" />
```

Comment:
I put the "style.css" file along with the other stylesheets into their own folder so they wouldn't clutter up the root directory. Not a big deal if you only have one stylesheet but it can be annoying when you have a bunch!